storagemarket/types/dealcheckpoints/checkpoints.go

```diff
@@ -10,6 +10,7 @@ const (
 	Published
 	PublishConfirmed
 	AddedPiece
+	Indexed
```

Comment:
I added the new checkpoint to streamline the UI. Otherwise, we will see the status in UI and client deal-status as "IndexedAndAnnounced" which might confuse folks when they chose not to announce. Let me know if that doesn't make sense and I will undo the checkpoint.

Reply:
Let's just change the string that we display in the UI to "Indexed".
I think it's better not to change the deal status because clients might already be relying on the name of the checkpoint.